src/app/components/footer/footer.component.html

```diff
@@ -3,13 +3,7 @@
     <div class="row">
       <div class="col-md-3 mt-md-0 mt-3">
         <div class="footer-widget d-flex justify-content-center">
-          <img
-            alt=""
-            class="img-fluid"
-            height="150"
-            src="assets/img/ICON/Logo Icon.png"
-            width="150"
-          />
+          <img alt="" class="img-fluid" height="150" src="assets/img/ICON/Logo Icon.png" width="150" />
```

Comment:
@CryceTruly It'd be nice to provide a value for `alt` attribute, just in case the image is not available.

Reply:
Thanks, @Victoradukwu, I have worked on this.